controller/proxy-injector/webhook.go

```diff
@@ -53,8 +62,23 @@ func Inject(api *k8s.API,
 		Allowed: true,
 	}
 
-	if !report.Injectable() {
-		log.Infof("skipped %s", report.ResName())
+	var parent *runtime.Object
+	if ownerRef := resourceConfig.GetOwnerRef(); ownerRef != nil {
+		objs, err := api.GetObjects(request.Namespace, ownerRef.Kind, ownerRef.Name)
+		if err != nil {
+			return nil, err
+		}
+		if len(objs) == 0 {
+			return nil, fmt.Errorf("couldn't retrieve parent object %s-%s-%s", request.Namespace, ownerRef.Kind, ownerRef.Name)
```

Comment:
do we want to just log this error and continue with injection, rather than returning? the `if parent != nil` checks below make me think we can gracefully handle this?

Reply:
Agreed :+1: